DuckDuckGo/BrowsingTips.swift

```diff
@@ -22,9 +22,9 @@ import Core
 
 protocol BrowsingTipsDelegate: NSObjectProtocol {
     
-    func showPrivacyGradeTip()
+    func showPrivacyGradeTip(didShow: @escaping (Bool) -> Void)
     
-    func showFireButtonTip()
+    func showFireButtonTip(didShow: @escaping (Bool) -> Void)
```

Comment:
Just a thought (no strong opinion about this): maybe instead of 'didShow', name it 'completion':
`showFireButtonTip(completion: @escaping (_ didShow: Bool)->Void)`

Reply:
I had completion originally, but it felt vague to me.

Reply:
👍
Yea, got the same feeling and I'm not sure I particularly like it even with `(_ didShow: Bool)->Void` as a clarification.